app/views/expenses/index.html.erb

```diff
@@ -20,6 +20,8 @@
         <td><%= exp.paid_on %></td>
         <td>
           <%= link_to t('expenses.index.edit'), edit_expense_path(exp.id), class: "btn btn-outline-primary"%>
+          <%= link_to t('expenses.index.destroy'), expense_path(exp.id), method: :delete,
+                      data: { confirm: "Are you sure?" }, class: "btn btn-outline-danger"%>
```

Comment:
Translations?

Reply:
I want to add modal from bootstrap for better view, but wherever i search i see only JavaScript script and HTML.